protocol/contracts/templegold/external/layerzero/oapp/OApp.sol

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: MIT
+
```

Comment:
Any reason for importing manually rather than just using the npm package?

https://docs.layerzero.network/v2/developers/evm/oft/quickstart#installation

`yarn add --dev @layerzerolabs/lz-evm-oapp-v2`

Reply:
We could also transfer ownership to a proxy contract, which has `ElevatedAccess`. And every admin call goes through `ElevatedAccess` which if forwarded to `TempleGold`